webapp/channels/src/components/at_sum_members_mention/notification_from_members_modal.tsx

```diff
@@ -91,13 +91,13 @@ function NotificationFromMembersModal(props: Props) {
         };
     });
 
-    const openDirectMessage = async (user: UserProfile) => {
+    const openDirectMessage = useCallback(async (user: UserProfile) => {
```

Comment:
This was actually not needed for this issue but I thought would be a nice improvement. I could not find where this component is being used so I could not test. Please test it on QA test